src/Files.App/Actions/Display/GroupAction.cs

```diff
@@ -1,137 +1,173 @@
 	}
 
 	internal class GroupByOriginalFolderAction : GroupByAction
 	{
-		protected override GroupOption GroupOption { get; } = GroupOption.OriginalFolder;
+		protected override GroupOption GroupOption
+			=> GroupOption.OriginalFolder;
 
-		public override string Label { get; } = "OriginalFolder".GetLocalizedResource();
+		public override string Label
+			=> "OriginalFolder".GetLocalizedResource();
 
-		public override string Description => "GroupByOriginalFolderDescription".GetLocalizedResource();
+		public override string Description
+			=> "GroupByOriginalFolderDescription".GetLocalizedResource();
 
-		protected override bool GetIsExecutable(ContentPageTypes pageType) => pageType is ContentPageTypes.CloudDrive;
+		protected override bool GetIsExecutable(ContentPageTypes pageType)
+			=> pageType is ContentPageTypes.CloudDrive;
 	}
 
 	internal class GroupByDateDeletedAction : GroupByAction
 	{
-		protected override GroupOption GroupOption { get; } = GroupOption.DateDeleted;
+		protected override GroupOption GroupOption
+			=> GroupOption.DateDeleted;
 
-		public override string Label { get; } = "DateDeleted".GetLocalizedResource();
+		public override string Label
+			=> "DateDeleted".GetLocalizedResource();
 
-		public override string Description => "GroupByDateDeletedDescription".GetLocalizedResource();
+		public override string Description
+			=> "GroupByDateDeletedDescription".GetLocalizedResource();
 
-		protected override bool GetIsExecutable(ContentPageTypes pageType) => pageType is ContentPageTypes.RecycleBin;
+		protected override bool GetIsExecutable(ContentPageTypes pageType)
+			=> pageType is ContentPageTypes.RecycleBin;
 	}
 
 	internal class GroupByFolderPathAction : GroupByAction
 	{
-		protected override GroupOption GroupOption { get; } = GroupOption.FolderPath;
+		protected override GroupOption GroupOption
+			=> GroupOption.FolderPath;
 
-		public override string Label { get; } = "FolderPath".GetLocalizedResource();
+		public override string Label
+			=> "FolderPath".GetLocalizedResource();
 
-		public override string Description => "GroupByFolderPathDescription".GetLocalizedResource();
+		public override string Description
+			=> "GroupByFolderPathDescription".GetLocalizedResource();
 
-		protected override bool GetIsExecutable(ContentPageTypes pageType) => pageType is ContentPageTypes.Library;
+		protected override bool GetIsExecutable(ContentPageTypes pageType)
+			=> pageType is ContentPageTypes.Library;
 	}
 
 	internal abstract class GroupByAction : ObservableObject, IToggleAction
 	{
-		protected IContentPageContext ContentContext { get; } = Ioc.Default.GetRequiredService<IContentPageContext>();
-		protected IDisplayPageContext DisplayContext { get; } = Ioc.Default.GetRequiredService<IDisplayPageContext>();
+		protected IContentPageContext ContentContext = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		protected IDisplayPageContext DisplayContext = Ioc.Default.GetRequiredService<IDisplayPageContext>();
```

Comment:
Can you move the assignment to the constructor?

Reply:
Fixed